third_party/pcsc-lite/src/src/hotplug_libusb.c

```diff
@@ -519,24 +495,23 @@ static void * HPEstablishUSBNotifications(int pipefd[2])
 		do_polling = true;
 	}
 
+	char dummy;
```

Comment:
this can be moved to `else` clause?

Reply:
We also use it below, in `write()`... But maybe I should try refactoring this into smaller functions.

Reply:
ack